docs/vnext/contract.md

```diff
@@ -1,8 +1,8 @@
-# Azure Deployments Extensibility Provider API Contract
+# Bicep Extension API Contract

```

Comment:
For context, the Bicep team has decided to adopt the term `extensions` (or `Bicep extensions` for added clarity) as providers is highly ambitious within the ARM context.